perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1777,6 +1779,27 @@ sub user_hide_everywhere : Private {
     $c->stash->{status_message} = _('That user’s reports and updates have been hidden.');
 }
 
+# Anonymize and remove name from all problems/updates, disable all alerts.
+# Remove their account's email address, phone number, password, etc.
+sub user_remove_account : Private {
+    my ( $self, $c, $user ) = @_;
+    $user->problems->update({ anonymous => 1, name => '' });
```

Comment:
Should this not also make sure that questionnaire sending is turned off for the problems? More to stop annoying bounces than anything else.

Reply:
Ooh, good point. Also moderation emails, thinking about it, hmm (probably rare enough not to bother there).